src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,16 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Car {
+    public static List<String> getCar(String cars) {
+        return new ArrayList<>(Arrays.asList(cars.split(",")));
+    }
```

Comment:
지금도 잘 해주셨지만, 좀더 간결하게 할 수 있을 것 같아요.

```suggestion
    public static List<String> getCar(String cars) {
        return Arrays.asList(cars.split(","));
    }
```